index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+  <div class="toolbar">
+    <header>
+      <a class="my-logo" href="#">
+        <img src="/images/My-Logo.png" alt="My Logo icon.">
+      </a>
+    </header>
+      <nav>
+        <a class="hamburger" href="#">
+          <img src="/images/Hamburger.png" alt="Hamburger icon.">
+        </a>
+      </nav>
+    </div>
+      <div class="background">
+    <main class="headline">
+      <h2 class="primarytext">I’m Marcos
+        Glad to see you!</h2>
+        <article class="primarytext2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</article>
+        <a class="connect" href="#">
+          <img src="/images/connect.png" alt="LET'S CONNECT icon">
+        </a>
```

Comment:
- [ ] I noticed that your "LET'S CONNECT" is an image. Please note that this call to action is not an image but should be a `<hx>` tag. Please make this change